README.md

```diff
@@ -14,7 +14,13 @@ The static marketing site for login.gov
 
 ## Development
 
-This is a Jekyll-built static site. To start serve the site locally in development:
+This is a Jekyll-built static site. To install dependencies:
+
+```
+bundle install
```

Comment:
I think he meant just say to use `make setup` instead. I can push a change in a minute